src/Config/Stage/Deploy.php

```diff
@@ -67,15 +41,14 @@ public function __construct(
      */
     public function get(string $name)
     {
```

Comment:
Add description to method that all values are already decoded and so on.

Reply:
added